CTP/common/src/com/navercorp/cubridqa/common/ConfigParameterConstants.java

```diff
@@ -0,0 +1,87 @@
+package com.navercorp.cubridqa.common;
+
+public class ConfigParameterConstants {
+	
+	//Prefix and suffix parameters
+	public static final String TEST_INSTANCE_PREFIX = "env.";
+	public static final String TEST_INSTANCE_HOST_SUFFIX = "ssh.host";
+	public static final String TEST_INSTANCE_PORT_SUFFIX = "ssh.port";
+	public static final String TEST_INSTANCE_USER_SUFFIX = "ssh.user";
+	public static final String TEST_INSTANCE_PASSWORD_SUFFIX = "ssh.pwd";
+	public static final String TEST_INSTANCE_HA_SLAVE_SUFFIX = "ssh.relatedhosts";
+	public static final String DEFAULT_CUBRID_PROPERTY_PREFIX = "default.cubrid";
+	public static final String DEFAULT_CUBRID_BROKER1_PROPERTY_PREFIX = "default.broker1";
+	public static final String DEFAULT_CUBRID_BROKER2_PROPERTY_PREFIX = "default.broker2";
+	
+	//Environment variables
+	public static final String CTP_SKIP_UPGRADE = "CTP_SKIP_UPGRADE";
+	public static final String CTP_BRANCH_NAME = "CTP_BRANCH_NAME";
+	
+	//CUBRID installation and configuration parameters
+	public static final String CUBRID_INSTALL_ROLE = "cubrid_install_role";
+	public static final String CUBRID_DOWNLOAD_URL = "cubrid_download_url";
+	public static final String CUBRID_ADDITIONAL_DOWNLOAD_URL = "cubrid_additional_download_url";
+	public static final String CUBRID_DB_CHARSET = "cubrid_db_charset";
+	public static final String CUBRID_CUBRID_PORT_ID = "cubrid.cubrid_port_id";
```

Comment:
as is:
public static final String CUBRID_CUBRID_PORT_ID = "cubrid.cubrid_port_id";
public static final String CUBRID_HA_PORT_ID = "ha.ha_port_id";
public static final String CUBRID_CM_PORT = "cm.cm_port";
public static final String CUBRID_BROKER1_BROKER_PORT = "broker1.BROKER_PORT";
public static final String CUBRID_BROKER2_BROKER_PORT = "broker2.BROKER_PORT";

to be:
public static final String ROLE_ENGINE="cubrid";
public static final String ROLE_HA="ha";
public static final String ROLE_CM="cm";
public static final String ROLE_BROKER1="broker1";
public static final String ROLE_BROKER2="broker2";

Then we have to implement more flexible policy like:

env.instance2.broker1.{any_property_name}={value}